packages/suite/src/utils/wallet/exportTransactionsUtils.ts

```diff
@@ -333,17 +334,17 @@ const prepareCsv = (
         line.push(v);
     });
 
-    lines.push(line.join(CSV_SEPARATOR));
+    lines.push(line.join(getCSVSeparator(locale)));
 
     // Prepare data
     content.forEach(item => {
         line = [];
 
         fieldKeys.forEach(field => {
-            line.push(sanitizeCsvValue(item[field]));
+            line.push(sanitizeCsvValue(item[field], locale));
         });
 
-        lines.push(line.join(CSV_SEPARATOR));
+        lines.push(line.join(getCSVSeparator(locale)));
```

Comment:
`getCSVSeparator` is called multiple times => 

```ts
const separator  = getCSVSeparator(locale)
```
```suggestion
        lines.push(line.join(separator));
```